src/ExtendedGridOrbitals.cc

```diff
@@ -250,7 +245,14 @@ void ExtendedGridOrbitals::initGauss(
     for (int icolor = 0; icolor < numst_; icolor++)
```

Comment:
This loop is over the columns of the tall-skinny matrix. Shouldn't the new copy between host and device be outside that loop? In particular if these copy involve the whole tall-skinny matrix, and not just one column.

Reply:
I've removed the `set` which has the wrong size anyway and I get the same result anyway. Since I copy the result back to the device in the same loop, I think this should be ok